src/main/java/uk/gov/pay/api/validation/PaymentSearchValidator.java

```diff
@@ -18,19 +18,33 @@
     public static final Set<String> VALID_STATES =
         new HashSet<>(Arrays.asList("created", "started", "submitted", "failed", "cancelled", "error", "confirmed", "captured"));
 
-    public static void validateSearchParameters(String state, String reference, String fromDate, String toDate) {
+    public static void validateSearchParameters(String state, String reference, String fromDate, String toDate, String pageNumber, String displaySize) {
         List<String> validationErrors = new LinkedList<>();
 
         validateState(state, validationErrors);
         validateReference(reference, validationErrors);
         validateFromDate(fromDate, validationErrors);
         validateToDate(toDate, validationErrors);
+        validatePage(pageNumber, validationErrors);
+        validateDisplaySize(displaySize, validationErrors);
 
         if (!validationErrors.isEmpty()) {
             throw new ValidationException(aPaymentError(SEARCH_PAYMENTS_VALIDATION_ERROR, join(validationErrors, ", ")));
         }
     }
 
+    private static void validatePage(String pageNumber, List<String> validationErrors) {
+        if (pageNumber!=null && Long.valueOf(pageNumber) < 1) {
```

Comment:
* It was a Long to start with thats why left with !=null, but isNotBlank makes sense with String.
* Integer now
* Will throw a NumberFormatException which is a Runtime exception if its not a number